src/components/badges/badges-style.ts

```diff
@@ -0,0 +1,21 @@
+import { colors } from '../../styles/colors';
+import { typography } from '../../styles/typography/typography';
+
+export const badgeStyles = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  border: `${typography.border.width} solid ${colors.NeutralMedium}`,
+  color: colors.NeutralXdark,
+  backgroundColor: colors.NeutralWhite,
+  width: '86.82px',
+  height: '47.05px',
```

Comment:
2 coisa: 

1. O height eu ate concordo que pode precisar definir, mas o width tb? Se o texto do badge for maior que 86px? Chegou a testar?
2. Esses valores quebrados fazem diferença? Por que não usar apenas o valor inteiro para facilitar?

Reply:
Lembra também que adotamos a convenção de usar múltiplos de 4px, pq não tem porque quebrar mais que isso, a menos em casos específicos como bordas ou linhas.